src/raft.c

```diff
@@ -1667,6 +1848,9 @@ void handleRedisCommand(RedisRaftCtx *rr,RaftReq *req)
      * commands we've received this as a RAFT.ENTRY input and bundling, probably through a
      * proxy, and bundling was done before.
      */
+
+    handleAsking(req);
```

Comment:
Why do you do this here and not inside `executeRaftRedisCommandArray()` ? 

Also, why do you serialize "ASKING" differently? Why not just leave it as it is and check inside `executeRaftRedisCommandArray()` like "MULTI" here: 

https://github.com/RedisLabs/redisraft/blob/e573dd291b76591e736d74d3eea1be059aac8bad/src/raft.c#L148